documents/motions/10.2019.4 - Quality Assurance Committee.md

```diff
@@ -0,0 +1,25 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 10.2019.4  
+**Subject:** WCA Quality Assurance Committee  
+**Intent:** Rights and duties of the WCA Quality Assurance Committee  
+**Submitted by:** Board of Directors  
+**Date:** June 1, 2019  
+
+# Motion
+
+The WCA Quality Assurance Committee is an Advisory Committee of the WCA with the role to support and promote continuous quality improvement within the WCA, as well as worldwide application of quality standards to ensure consistent high quality of processes and quality awareness among WCA Staff, Regional Organizations, Competition Organizers, and Competition Staff.
+
+The WCA Quality Assurance Committee has the following rights and duties:
+
+1. Maintaining worldwide quality standards:
+   1. Promoting awareness and enforcing application of these standards among WCA Staff, Regional Organizations, Competition Organizers, and Competition Staff.
+2. Managing the processes of continuous quality improvement:
+   1. Evaluating operations of the WCA to discover areas for possible quality improvements.
+   2. Proposing quality improvements and handling quality improvement proposals by others.
+      1. Any proposal(s) must include conclusive reasoning about a group of persons or a wider region that they apply to. Individual complaints may be rejected without further examination by the Committee, at the discretion of the Leader.
+   3. Supporting the implementation of quality improvements.
+3. Managing the processes of quality reporting:
+   1. Producing periodic quality assessment reports on the quality, development, distribution, and other characteristics of the operations of the WCA and of WCA Competitions.
+4. Periodically or spontaneously examining and verifying the knowledge and competences of WCA Staff that is necessary to operate within the previously defined quality standards.
```

Comment:
If there will be periodic or spontaneous examination of WCA Staff then should we also add a clause stating that all staff are obliged to cooperate with such examinations?

Reply:
Do you think 5.4 of the Delegates Motion is enough?
5.4 WCA Delegates shall sustain knowledge and skills regarding the Mission, Spirit, and
Regulations of the WCA.

Reply:
I don't think it's enough, as them not wanting to cooperate with such examinations doesn't necessarily mean that they don't sustain the required knowledge and skill.

Reply:
Okay, I think it's fine to add that as a bullet.

Reply:
I agree with adding this point specifically.

Reply:
I have already included this in a `Staff review` commit that I'll be pushing live later.